Resources/config/doctrine/model/AccessToken.orm.xml

```diff
@@ -14,6 +14,11 @@
         <field name="userIdentifier" type="string" length="128" nullable="true" />
         <field name="scopes" type="oauth2_scope" nullable="true" />
         <field name="revoked" type="boolean" />
+        <indexes>
+            <index name="idx_oauth2_access_token_client" columns="client"/>
+            <index name="idx_oauth2_access_token_userIdentifier" columns="userIdentifier"/>
```

Comment:
@HypeMC do you remember why we've reverted this index in the past? There were some issues with Doctrine index naming IIRC.

Reply:
@X-Coder264 Yep, Doctrine didn't take into account the naming strategy https://github.com/trikoder/oauth2-bundle/pull/197 , this needs to be removed for now.

Reply:
@X-Coder264 @HypeMC I don't understand what's the problem with this? The only bug I found is you can't have more indexes with the same name, even on different tables.

Reply:
@franjo-zadelj-trikoder https://github.com/trikoder/oauth2-bundle/issues/196

Reply:
@franjo-zadelj-trikoder Please keep in mind that this is an open source bundle used by a lot of projects, not just our internal ones. Though everything might work in our case, we can't expect that everyone uses the same naming strategy as we do. To ensure compatibility with different naming strategies on different projects we must not add this for now. Once the bug gets fixed, we can consider adding it back.